src/app/page.tsx

```diff
@@ -0,0 +1,339 @@
+                                  <path d="M21 21C21 21.7956 20.6839 22.5587 20.1213 23.1213C19.5587 23.6839 18.7956 24 18 24H6C5.20435 24 4.44129 23.6839 3.87868 23.1213C3.31607 22.5587 3 21.7956 3 21V3C3 2.20435 3.31607 1.44129 3.87868 0.87868C4.44129 0.316071 5.20435 0 6 0L14.25 0L21 6.75V21ZM6 1.5C5.60218 1.5 5.22064 1.65804 4.93934 1.93934C4.65804 2.22064 4.5 2.60218 4.5 3V18L7.836 14.664C7.95422 14.5461 8.10843 14.4709 8.27417 14.4506C8.43992 14.4302 8.60773 14.4657 8.751 14.5515L12 16.5L15.2355 11.97C15.2988 11.8815 15.3806 11.8078 15.4753 11.754C15.5699 11.7003 15.6751 11.6678 15.7836 11.6588C15.8921 11.6498 16.0012 11.6645 16.1034 11.702C16.2056 11.7394 16.2985 11.7986 16.3755 11.8755L19.5 15V6.75H16.5C15.9033 6.75 15.331 6.51295 14.909 6.09099C14.4871 5.66903 14.25 5.09674 14.25 4.5V1.5H6Z" fill="#202945" />
+                                </svg>
+                              </div>
+                              <span className="text-black text-center md:ml-4 ml-2 md:text-base text-xs">Passport back.png</span>
+                              <span className="text-black md:ml-4 ml-2 md:text-base text-xs text-center">Preview</span>
+                            </div>
+                            <span className="text-black md:text-base text-xs">5.7 MB</span>
+                          </div>
+                        </div>
+                        <div className="relative w-full mt-6">
+                          <label className="block mb-2 text-sm font-medium text-gray-900 absolute z-10 bottom-[1.8rem] bg-white left-4 px-2">Billing address</label>
+                          <input type="email" name="email" id="email" className="w-full border border-gray-300 text-black text-start p-[0.7rem] px-8 rounded-lg" placeholder="name@company.com" />
+                        </div>
+                        <div className="flex justify-between w-full mt-6">
+                          <div className="relative w-[48%]">
+                            <label className="block absolute px-2 text-black bottom-[2.7rem] z-10 bg-white left-4 text-sm font-medium">Adults passengers</label>
+                            <select id="countries" className="border border-gray-300 text-gray-900 text-sm rounded-lg block w-full p-4 ">
+                              <option selected>1</option>
+                              <option value="US">2</option>
+                              <option value="CA">3</option>
+                              <option value="FR">4</option>
+                              <option value="DE">5</option>
+                            </select>
+                          </div>
+                          <div className="relative w-[48%]">
+                            <label className="block px-2 absolute text-black bottom-[2.7rem] z-10 bg-white left-4 text-sm font-medium">Kids passengers</label>
```

Comment:
I feel we are abusing of tailwind CSS, we can create own CSS for really global things like the font-types

Reply:
sure